make/alerts.mk

```diff
@@ -1,12 +1,9 @@
 ##@ Alerts
-export WORKDIR ?= $(shell pwd)
 export IMAGE ?= quay.io/prometheus/prometheus
-export AVAILABILITY_SLO_RULES ?= $(WORKDIR)/examples/alerts/slo-availability.yaml
```

Comment:
@R-Lawton you are the original owner of this script. heads up as I introduced small changes.

on a different note, why exporting makefile variables? are they being used in sub-processes? If not, they are not required.